cloudigrade/account/tasks.py

```diff
@@ -60,6 +62,26 @@ def create_volume(ami_id, snapshot_id):
     """
     zone = settings.HOUNDIGRADE_AWS_AVAILABILITY_ZONE
     volume_id = aws.create_volume(snapshot_id, zone)
-    # TODO Replace the next `print` call with a call to the next task.
-    # Do something like: enqueue_ready_volume(ami_id, volume_id)
-    print('Volume {} is being created for AMI {}'.format(volume_id, ami_id))
+
+    enqueue_ready_volume.delay(ami_id, volume_id, zone[:-1])
```

Comment:
This it probably fine for now, but the chopping of the last character feels a little bit tech-debt-y since it presumes there will only be 9 of zones in a given region. Currently, US East is the biggest with 6 zones.

Reply:
Nah, it was laziness. I'll use `describe_availability_zones` via boto3/EC2 and pull out the region from that.